logback-android/build.gradle

```diff
@@ -47,13 +47,13 @@ android {
 
 dependencies {
 
-    testImplementation('junit:junit:4.12') {
+    testImplementation('junit:junit:4.13.2') {
         exclude(module: 'hamcrest-core')
     }
 
     testImplementation 'org.hamcrest:hamcrest-junit:2.0.0.0'
-    testImplementation 'org.robolectric:robolectric:4.3.1'
-    testImplementation 'org.mockito:mockito-core:2.23.4'
+    testImplementation 'org.robolectric:robolectric:4.8.1'
+    testImplementation 'org.mockito:mockito-core:4.7.0'
```

Comment:
I plan on updating all deps separately. If they're needed for this PR, please explain why.